File: Php/uberBlack.php
<?php

require_once("car.php");

class UberBlack extends Car{
    public $typeCarAcepted;
    public $seatsMaterial;

    public function __construct($license, Account $driver, $typeCarAcepted, $seatsMaterial){
        parent::__construct($license, $driver);
        $this->typeCarAcepted = $typeCarAcepted;
        $this->seatsMaterial = $seatsMaterial;
    }
        
}
?>

File: Java/UberBlack.java
package Java;

import java.util.ArrayList;
import java.util.Map;

public class UberBlack extends Car {

    Map<String, Map<String, Integer>> typeCarAcepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String licence, Account driver, Map<String, Map<String, Integer>> typeCarAcepted, ArrayList<String> seatsMaterial) {
        super(licence, driver);
        
        this.typeCarAcepted = typeCarAcepted;
        this.seatsMaterial = seatsMaterial;
    }
    
}
